acceptance/config/beaker/options.rb

```diff
@@ -11,5 +11,5 @@
  "puppetserver-confdir"=>"/etc/puppetlabs/puppetserver/conf.d",
  "puppetserver-config"=>
   "/etc/puppetlabs/puppetserver/conf.d/puppetserver.conf",
- :puppet_version=>"5.2.0.33.g571b94f",
- :puppet_build_version=>"b9b9d3f893abf6d60aa20558816341d6794351cf"}
+ :puppet_version=>"5.2.0.68.g384311c"
```

Comment:
This is missing a trailing comma. which is causing beaker to fail:

```
bundler: failed to load command: beaker
SyntaxError: (eval):15: syntax error, unexpected tSYMBEG, expecting '}'
 :puppet_build_version=>"b1172c7f8c93541c39f060e114062ad16227201a"}
  ^
```

Reply:
Whoops, updated.